lib_eio_linux/eio_linux.ml

```diff
@@ -917,6 +917,45 @@ module Low_level = struct
     Eio_unix.run_in_systhread @@ fun () ->
     Unix.getaddrinfo node service []
     |> List.filter_map to_eio_sockaddr_t
+
+    module Process = struct
+      external pidfd_open : int -> Unix.file_descr = "caml_eio_pidfd_open" 
+
+      type t = {
+        process : FD.t;
+        pid : int;
+        mutable hook : Switch.hook option;
```

Comment:
You can avoid this option with `Eio.Switch.null_hook`.

Reply:
https://github.com/ocaml-multicore/eio/pull/435/commits/ab5a1faf142c04c5dd6ddf4807e5b479fd7b26c0